pstress/generate_mysqld_options.sh

```diff
@@ -0,0 +1,569 @@
+    echo "${OPTION}=1" >> ${OUTPUT_FILE}
+    echo "${OPTION}=2" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--csv-mode" ]; then
+    echoit "  > Adding possible values IETF_QUOTES for option '${OPTION}' to the final list..."
+    echo "${OPTION}=IETF_QUOTES" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-log-files-in-group" ]; then
+    echoit "  > Adding possible values 0,1,2,5,10 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=0" >> ${OUTPUT_FILE}
+    echo "${OPTION}=1" >> ${OUTPUT_FILE}
+    echo "${OPTION}=2" >> ${OUTPUT_FILE}
+    echo "${OPTION}=5" >> ${OUTPUT_FILE}
+    echo "${OPTION}=10" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--log-warnings-suppress" ]; then
+    echoit "  > Adding possible values 1592 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=1592" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--slave-type-conversions" ]; then
+    echoit "  > Adding possible values ALL_LOSSY, ALL_NON_LOSSY for option '${OPTION}' to the final list..."
+    echo "${OPTION}=ALL_LOSSY" >> ${OUTPUT_FILE}
+    echo "${OPTION}=ALL_NON_LOSSY" >> ${OUTPUT_FILE}
+    echo "${OPTION}=ALL_SIGNED" >> ${OUTPUT_FILE}
+    echo "${OPTION}=ALL_UNSIGNED" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-checksum-algorithm" ]; then
+    echoit "  > Adding possible values innodb, crc32 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=innodb" >> ${OUTPUT_FILE}
+    echo "${OPTION}=crc32" >> ${OUTPUT_FILE}
+    echo "${OPTION}=none" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-cleaner-lsn-age-factor" ]; then
+    echoit "  > Adding possible values legacy, high_checkpoint for option '${OPTION}' to the final list..."
+    echo "${OPTION}=legacy" >> ${OUTPUT_FILE}
+    echo "${OPTION}=high_checkpoint" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-corrupt-table-action" ]; then
+    echoit "  > Adding possible values assert, warn for option '${OPTION}' to the final list..."
+    echo "${OPTION}=assert" >> ${OUTPUT_FILE}
+    echo "${OPTION}=warn" >> ${OUTPUT_FILE}
+    echo "${OPTION}=salvage" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-empty-free-list-algorithm" ]; then
+    echoit "  > Adding possible values legacy, backoff for option '${OPTION}' to the final list..."
+    echo "${OPTION}=legacy" >> ${OUTPUT_FILE}
+    echo "${OPTION}=backoff" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-file-format-max" ]; then
+    echoit "  > Adding possible values Antelope, Barracuda for option '${OPTION}' to the final list..."
+    echo "${OPTION}=Antelope" >> ${OUTPUT_FILE}
+    echo "${OPTION}=Barracuda" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-foreground-preflush" ]; then
+    echoit "  > Adding possible values sync_preflush, exponential_backoff for option '${OPTION}' to the final list..."
+    echo "${OPTION}=sync_preflush" >> ${OUTPUT_FILE}
+    echo "${OPTION}=exponential_backoff" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-buffer-pool-evict" ]; then
```

Comment:
remove this variable as it is a debug-only variable. There is no documentation for this variable as well